File: ServerApplication/entities/Master.java
package com.example.test1.entities;

import java.util.*;

public class Master {
    private String name;
    private String birthday;
    private List<Cat> cats;
    private long id;

    public Master(String newName, String newBirthday) {
        this.name = newName;
        this.birthday = newBirthday;
        cats = new ArrayList<Cat>();
        this.id = (long) (Math.random() * 1000);
    }
    public String getName() {
        return this.name;
    }

    public String getBirthday() {
        return this.birthday;
    }
    public List<Cat> getCats() {
        return this.cats;
    }

    public long getId() {
        return this.id;
    }

    public void setNewName(String newName) {
        this.name = newName;
    }
    public void setCats(List<Cat> cats) {
        this.cats = cats;
    }

    public void setBirthday(String birthday) {
        this.birthday = birthday;
    }

    public void setId(long id) {
        this.id = id;
    }

    public void addCat(Cat newCat) {
        this.cats.add(newCat);
    }

    public void CreateMaster(Master master) {
        this.name = master.name;
        this.birthday = master.birthday;
        this.id = master.id;
        this.cats = master.cats;
    }

}

File: lab-2/lab2/Master.cs
using System.Collections.Generic;

namespace Lab2
{
    public class Master
    {
        public string name;
        public string birthday;
        public List<Cat> cats;
        public long id;
        public Master(string newName, string newBirthday)
        {
            name = newName;
            birthday = newBirthday;
        }

        public string Getname()
        {
            return name;
        }

        public void Setname(string newValue)
        {
            name = newValue;
        }

        public string Getbirthday()
        {
            return birthday;
        }

        public void Setbirthday(string newValue)
        {
            birthday = newValue;
        }

        public List<Cat> Getcats()
        {
            return cats;
        }

        public void Setcats(List<Cat> newValue)
        {
            cats = newValue;
        }

        public long Getid()
        {
            return id;
        }

        public void Setid(long newValue)
        {
            id = newValue;
        }
    }
}
